src/main/java/io/cdap/plugin/dlp/SensitiveRecordRedaction.java

```diff
@@ -288,6 +300,8 @@ private DlpServiceSettings getSettings() throws IOException {
     @Macro
     @Nullable
     private String templateId;
+
+
     private static final Gson gson = new GsonBuilder()
```

Comment:
static final variable name should be ALL_CAPS (`GSON` in this case).

Reply:
done